web/satellite/src/components/accessGrants/CreateAccessModal.vue

```diff
@@ -809,15 +816,22 @@ export default class CreateAccessModal extends Vue {
                 await this.$store.dispatch(ACCESS_GRANTS_ACTIONS.GET_GATEWAY_CREDENTIALS, {accessGrant: this.access});
 
                 await this.$notify.success('Gateway credentials were generated successfully');
-
+                
                 await this.analytics.eventTriggered(AnalyticsEvent.GATEWAY_CREDENTIALS_CREATED);
 
                 this.areKeysVisible = true;
             } catch (error) {
                 await this.$notify.error(error.message);
             }
         }
+        else if (this.checkedType === 'api') {
```

Comment:
nit - to match the convention elsewhere in the project, put the `else if` on the same line as the previous curly brace. e.g. 
```
if {
...
} else if {
...
}
```